web-bluetooth/read-characteristic-value-changed.js

```diff
@@ -1,13 +1,34 @@
+var bluetoothDevice;
 var batteryLevelCharacteristic;
 
 function onReadBatteryLevelButtonClick() {
-  log('Requesting Bluetooth Device...');
-  navigator.bluetooth.requestDevice(
-    {filters: anyDevice(), optionalServices: ['battery_service']})
-  .then(device => {
-    log('Connecting to GATT Server...');
-    return device.gatt.connect();
+  let requestDevice = Promise.resolve();
```

Comment:
I've added a button to reset device not switch as I didn't want to add more complexity.